app/src/main/java/com/keylesspalace/tusky/viewdata/TrendingViewData.kt

```diff
@@ -0,0 +1,37 @@
+/* Copyright 2023 Tusky Contributors
+ *
+ * This file is a part of Tusky.
+ *
+ * This program is free software; you can redistribute it and/or modify it under the terms of the
+ * GNU General Public License as published by the Free Software Foundation; either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Tusky is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even
+ * the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General
+ * Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with Tusky; if not,
+ * see <http://www.gnu.org/licenses>. */
+
+package com.keylesspalace.tusky.viewdata
+
+import com.keylesspalace.tusky.entity.TrendingTag
+
+/**
+ * Created by charlag on 11/07/2017.
+ *
+ * Class to represent data required to display either a notification or a placeholder.
+ * It is either a [TrendingViewData.Concrete] or a [TrendingViewData.Placeholder].
+ */
+sealed class TrendingViewData {
```

Comment:
This is probably obsolete, right, as there are no placeholders in the trending API?

Reply:
Yes, the comment was wrong. However, I was planning on using the TrendingViewData as a parent for the other Trending data too. In order to use the same adapter.